src/syngen/ml/data_loaders/data_loaders.py

```diff
@@ -318,3 +319,48 @@ def load_data(self, path: str, **kwargs) -> Tuple[pd.DataFrame, None]:
     def save_data(self, path: str, data, **kwargs):
         with open(path, "wb") as f:
             pickle.dump(data, f)
+
+
+class ExcelLoader:
+    """
+    Class for loading and saving data in Excel format
+    """
+    def __init__(self):
+        self.format = get_context().get_config()
+        self.sheet_name = self.format.get("sheet_name", 0)
+        self.format = {
+            k: v for k, v in self.format.items()
+            if k in ExcelFormatSettingsSchema._declared_fields.keys()
+        }
+
+    def _load_data(self, path: str) -> Tuple[pd.DataFrame, Dict]:
+        """
+        Load data in Excel format
+        """
+        try:
+            df = pd.read_excel(path, sheet_name=self.sheet_name)
+            if isinstance(self.sheet_name, list) or self.sheet_name is None:
+                dfs = [df for sheet_name, df in df.items()]
+                df = pd.concat(dfs, ignore_index=True)
+            global_context({})
+            return df, CSVConvertor({"fields": {}, "format": "CSV"}, df).schema
+        except FileNotFoundError as error:
+            message = f"It seems that the path to the table isn't valid.\n" \
+                      f"The details of the error - {error}.\n" \
+                      f"Please, check the path to the table"
+            logger.error(message)
```

Comment:
Please add the original exception to the log.

Reply:
Yes, sure. Thank you for your comment